### PR TITLE
add prometheus client dependencies to setup

### PR DESCRIPTION
**Pull request by shaisgv**

### CODE REVIEW

**Comment by tzuryby** [comment]
where is this coming from?
not sure I am willing to "play" with curieconf at this moment, unless it is a necessity

**Comment by shaisgv** [comment]
@tzuryby, this was requested by Evegeniy,.
@Freakachoo , can you please answer Zuri.

**Comment by Freakachoo** [comment]
@tzuryby we're using Prometheus to export flask metrics. 
Here:
https://github.com/curiefense/curiefense/blob/main/curiefense/curieconf/server/curieconf/confserver/__init__.py#L9
But this module wasn't added to dependencies. So, here we are just adding it.

File: curiefense/curieconf/server/setup.py [hunk 1]
```diff
@@ -38,6 +38,7 @@
         "fasteners",
         "jsonpath-ng==1.5.3",
         "pydash==5.0.2",
+        "prometheus-flask-exporter==0.20.3",
```
Comment (Freakachoo) [inline curiefense/curieconf/server/setup.py:41]: @shaisgv seems like it has different indentation

**Pull request closed without merging**